spec/models/aggregated_cash_outgoings_spec.rb

```diff
@@ -0,0 +1,457 @@
+require 'rails_helper'
+
+RSpec.describe AggregatedCashOutgoings, type: :model do
+  let(:aco) { described_class.new }
+  let(:application) { create :legal_aid_application }
+  let(:categories) { %i[rent_or_mortgage maintenance_out] }
+  let!(:rent_or_mortgage) { create :transaction_type, :rent_or_mortgage }
+  let!(:maintenance_out) { create :transaction_type, :maintenance_out }
+  let!(:child_care) { create :transaction_type, :child_care }
+  let!(:legal_aid) { create :transaction_type, :legal_aid }
+
+  describe '#find_by' do
+    context 'no cash income transaction records' do
+      it 'returns an empty model' do
+        aco = described_class.find_by(legal_aid_application_id: application.id)
+        expect(aco.check_box_rent_or_mortgage).to be_nil
+        expect(aco.rent_or_mortgage1).to be_nil
+        expect(aco.rent_or_mortgage2).to be_nil
+        expect(aco.rent_or_mortgage3).to be_nil
+        expect(aco.check_box_child_care).to be_nil
+        expect(aco.child_care1).to be_nil
+        expect(aco.child_care2).to be_nil
+        expect(aco.child_care3).to be_nil
+        expect(aco.check_box_maintenance_out).to be_nil
+        expect(aco.maintenance_out1).to be_nil
+        expect(aco.maintenance_out2).to be_nil
+        expect(aco.maintenance_out3).to be_nil
+        expect(aco.check_box_legal_aid).to be_nil
+        expect(aco.legal_aid1).to be_nil
+        expect(aco.legal_aid2).to be_nil
+        expect(aco.legal_aid3).to be_nil
+      end
+    end
+
+    context 'cash income transaction records exist' do
+      let(:maintenance_out1) { create :cash_transaction, :credit_month_1, legal_aid_application: application, transaction_type: maintenance_out }
```

Comment:
if you could, maybe change the factory for cash_transaction around credit_month_1 to month_1, month_2, month_3
as it doesn't look to be declaring whether or not it's credit or debit.

It looks like transaction_type: ... is the one deciding whether its credit or debit.